client.py

```diff
@@ -51,11 +51,11 @@ def parse_ServerResponse(self, response):
         assert x.tag == '{http://schemas.xmlsoap.org/soap/envelope/}Envelope'
         r = x.find('.//{http://ctkipservice.rsasecurity.com}' + outer)
         ad = r.find('{http://ctkipservice.rsasecurity.com}AuthData')
-        assert ad.text == self.auth == response.headers.get('Authorization')
+        #assert ad.text == self.auth == response.headers.get('Authorization')
```

Comment:
My server doesn't send `Authorization` either, but for me the `ad.text == self.auth` assertion still held.  Does it not for you?